rusk/src/lib/lib.rs

```diff
@@ -227,12 +227,17 @@ impl Rusk {
     }
 
     /// Accept the given transactions.
+    ///
+    ///   * `consistency_check` - represents a state_root, the caller expects to
+    ///   be returned on successful transactions execution. Passing a zero-ed
+    ///   value disables the check.
```

Comment:
Do we have the need now to disable the check? If not, I'm for removing that.
Anyway, I'm for using an Option instead of empty array

Reply:
Updated it to an Option.
in case we need these methods in any unit tests or mocking, consistency_check shouldn't be mandatory.

Reply:
Actually, making it Option failed unit tests. I'll rollback to zero-ed value

```
2023-07-19T13:29:51.602255Z  INFO rusk::vm: Received accept request
thread 'services::stake::stake' panicked at 'generator procedure to succeed: Cannot execute txs: Inconsistent state root 0ca00d8466f08acadd382c06a069d4c9e68ac1a0d55a6c4a2f97537f434680f3!!', rusk/tests/services/stake.rs:65:10
test services::stake::stake ... FAILED
```

Reply:
Weird... then let's keep it as is for now